llvm/include/llvm/CodeGen/MachineFrameInfo.h

```diff
@@ -280,7 +280,7 @@ class MachineFrameInfo {
   /// setup/destroy pseudo instructions (as defined in the TargetFrameInfo
   /// class).  This information is important for frame pointer elimination.
   /// It is only valid during and after prolog/epilog code insertion.
-  unsigned MaxCallFrameSize = ~0u;
+  unsigned long MaxCallFrameSize = ~0ul;
```

Comment:
uint64_t ?

Reply:
I wasn't entirely sure when to use `unsigned long`/`long` instead of `uint64_t`/`int64_t` so I tried to follow the convention of the code I was changing. 

Is there a preferred style I should be using across the whole patch or would you prefer I just update this instance?

Reply:
I'd prefer explicit fixed bitwidth integer types in structs wherever possible tbh

Reply:
Updated!